### PR TITLE
JS import maps: first draft

### PR DESCRIPTION
**Pull request by sanderborgman**
Article about Import maps in Javascript

### CODE REVIEW

File: data/authors/sander-borgman.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Sander Borgman
+avatar: /authors/io.jpg
```
Comment (sanderdejong88) [inline data/authors/sander-borgman.md]: Missing your own avatar here :)

File: data/authors/sander-borgman.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+email: sander.borgman@iodigital.com
```
Comment (sanderdejong88) [inline data/authors/sander-borgman.md]: email isn't a field we use

File: data/blog/js-import-maps.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+---
+title: 'Import maps in JavaScript'
+date: '2023-04-25'
+tags: ['frontend', 'web-capabilities', 'javascript']
+images: ['/articles/project-fugu/header.webp']
```
Comment (sanderdejong88) [inline data/blog/js-import-maps.md]: if you can't find a good image you can also do a post without image

File: data/blog/js-import-maps.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    }
+  }
+</script>
+```
+
+In this example we have defined a scope for the `/legacy/` path. For any modules loaded from this path we are importing an older version of Moment.js.
+
+## Can I use them already?
+
+It's important to note that import maps are still a relatively new technology and may not be suitable for all projects. However, they are supported in most recent versions of the major browsers: Chrome, Firefox, and Safari. Let's explore some use cases.
```
Comment (sanderdejong88) [inline data/blog/js-import-maps.md]: Might be nice to link caniuse: https://caniuse.com/import-maps

File: data/blog/js-import-maps.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+```html
+<script type="module">
+  import moment from 'moment'
+  // ...
+</script>
+```
+
+Without a bundler, the browser will not be able to resolve the `moment` package.
+
+In this blog post, we'll explore import maps to solve this problem and show you how to get started with this powerful tool for managing JavaScript dependencies.
```
Comment (sanderdejong88) [inline data/blog/js-import-maps.md:21]: Maybe link the MDN docs? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap

File: data/blog/js-import-maps.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+        "imports": {
+          "moment": "https://cdn.jsdelivr.net/npm/moment@2.18.1/moment.js"
+        }
+      }
+    }
+  }
+</script>
+```
+
+In this example we have defined a scope for the `/legacy/` path. For any modules loaded from this path we are importing an older version of Moment.js.
```
Comment (sanderdejong88) [inline data/blog/js-import-maps.md:108]: Another option would be to create 2 maps for moment:
```
"imports": {
      "moment@2.29": "https://cdn.jsdelivr.net/npm/moment@2.29.4/moment.js",
      "moment@2.18": "https://cdn.jsdelivr.net/npm/moment@2.18.1/moment.js"
    },
```
This way you're not limited to scope an entire folder in your code.

File: data/blog/js-import-maps.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+## Can I use them already?
+
+It's important to note that import maps are still a relatively new technology and may not be suitable for all projects. However, they are supported in most recent versions of the major browsers: Chrome, Firefox, and Safari. Let's explore some use cases.
+
+### Prototyping
+
+For prototyping and during development, import maps can be an excellent choice as they allow developers to quickly experiment with different libraries and packages without the need for a build process. This can be useful in the early stages of a project when you're exploring different options and need to move quickly.
+
+### Buildless development
```
Comment (sanderdejong88) [inline data/blog/js-import-maps.md]: Might be nice to mention some of the packages available. E.g. the vite plugin for it.

Comment (sanderborgman) [reply data/blog/js-import-maps.md]: Great idea! Added a section for frontend tooling.

**Pull request merged**